DFC.Api.Content.UnitTests/Files/JobProfileRecordResponse_1.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "ncs__SalaryStarter": "12000",
```

Comment:
Just as a note to anyone else looking - we plan on removing the ncs__ property prefix within the graph itself (not as part of this api).